### PR TITLE
[CI:DOCS] Fix up example description of podman-pod commands

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podman-pod-exists.1.md [hunk 1]
```diff
@@ -14,20 +14,18 @@ was an issue accessing the local storage.
...
+Check if a pod called `web` exists in local storage (the pod does actually exist):
 ```
 $ sudo podman pod exists web
 $ echo $?
 0
-$
 ```
 
-Check if a pod called `backend` exists in local storage (the pod does not actually exist).
+Check if a pod called `backend` exists in local storage (the pod does not actually exist):
```
Comment (vrothberg) [inline docs/source/markdown/podman-pod-exists.1.md:24]: I find it a bit inconsistent that we're using an article for `pod` but not for `local storage`.  Shall we use "the local storage" in all cases?

**Review by vrothberg — APPROVED** [review]
LGTM

File: docs/source/markdown/podman-pod-exists.1.md (hunk 1 above)
Comment (edsantiago) [reply docs/source/markdown/podman-pod-exists.1.md]: I think "local storage" is the generally-accepted term; like "system storage" (would sound weird with an article)

**Comment by edsantiago** [comment]
LGTM

**Pull request merged**